packages/wallets/src/utils/cfdUtils.ts

```diff
@@ -9,3 +9,8 @@ export const platformPasswordResetRedirectLink = (platform: TPlatforms.All, isVi
             return isVirtual ? 21 : 20;
     }
 };
+
+export const platformMapping: Record<string, Exclude<TPlatforms.All, 'ctrader'>> = {
```

Comment:
Dont be needing this. Mapping the api doesnt make sense. Can just use the url to determine the platform